Library/Homebrew/cmd/info.rb

```diff
@@ -239,7 +239,7 @@ def info_formula(f, args:)
 
     if stable = f.stable
       s = "stable #{stable.version}"
-      s += " (bottled)" if stable.bottled?
+      s += " (bottled)" if stable.bottled? && (stable.bottle_specification.prefix == ENV["HOMEBREW_PREFIX"])
```

Comment:
Shouldn't this be fixed in the `#bottled?` method itself?

Reply:
That's a good question: idk. I don't have a preference as to where we fix this.

Reply:
And by the way, feel free to submit a PR that would do that 👍 I won't have time for this in the next few hours.